.circleci/config.yml

```diff
@@ -31,6 +43,62 @@ jobs:
       - run: yarn test --ci --runInBand
       - run: bash <(curl -s https://codecov.io/bash)
 
+  install-e2e-dependencies:
+    <<: *e2e-defaults
```

Comment:
Doesn't seem to need to startup redis/postgres for the prepare job, which seems to take up >1 minute https://circleci.com/gh/yunity/karrot-frontend/4232

Reply:
True. Although I think displayed times are not accurate because it also counts in the time until the required steps are ready and also if it's limited by the number of VMs (max 4 in parallel).